### DISCUSSION TITLE
Try to plot the data but it shows following error. Please help!

### DISCUSSION THREAD

**Discussion by Arslaan4341** [insarlab/MintPy#994] [General]
I am trying to plot my results, but I get the following error-
![issue](https://user-images.githubusercontent.com/110322003/234063210-648db2dd-a0ff-43ab-99fa-b2f7d0339aba.JPG)

**Comment by yunjunz** [comment]
Check the updated API usage here: https://github.com/insarlab/MintPy/discussions/913
